tests/api/PDFDocument.spec.ts

```diff
@@ -303,15 +303,65 @@ describe(`PDFDocument`, () => {
     });
   });
 
-  describe(`setTitle() method with options`, () => {
+  describe(`viewerPreferences`, () => {
     it(`defaults to an undefined ViewerPreferences dict`, async () => {
       const pdfDoc = await PDFDocument.create();
 
       expect(
         pdfDoc.catalog.lookupMaybe(PDFName.of('ViewerPreferences'), PDFDict),
       ).toBeUndefined();
     });
+    it(`can get/set HideToolbar, HideMenubar, HideWindowUI, FitWindow, CenterWindow, DisplayDocTitle, NonFullScreenPageMode, Direction, PrintScaling, Duplex, PickTrayByPDFSize, PrintPageRange, NumCopies from a new document`, async () => {
+      const pdfDoc = await PDFDocument.create();
```

Comment:
This test is good. Please add a second test that verifies the viewer preferences can be read from an existing PDF document (with `PDFDocument.load(...)`) that already has viewer preferences defined. You'll need to create a new document and adjust all of its viewer preferences in Adobe Acrobat (or another PDF editor that's capable of changing all these properties). Be sure to commit the PDF to the `assets/pdfs` directory as well.